cumulusci/robotframework/Salesforce.py

```diff
@@ -15,13 +15,15 @@
 from SeleniumLibrary.errors import ElementNotFound, NoOpenBrowser
 from urllib3.exceptions import ProtocolError
 
+from .salesforce_robot_library_base import SalesforceRobotLibraryBase
+
 OID_REGEX = r"^(%2F)?([a-zA-Z0-9]{15,18})$"
 
 lex_locators = {}  # will be initialized when Salesforce is instantiated
 
 
 @selenium_retry
-class Salesforce(object):
+class Salesforce(SalesforceRobotLibraryBase):
```

Comment:
It seems non-ideal for the Salesforce library to have a `salesforce` property that returns an instance of itself.

Reply:
I suppose so, but I guess the root cause of the weirdness is using this unusual dynamic import technique. I don't know why we import code this way.

Anyhow, I can fix it by removing the base class usage from Salesforce.py or by removing this helper here.

Reply:
What do you mean by "this unusual dynamic import technique"? Are you referring to the properties? It's not importing anything, it's simply providing a shortcut for commonly used objects.